docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: server-app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    environment:
+      - ENV=local
```

Comment:
In code doesn't use the variable `ENV`, so it is useless and can be removed. Also need to add `PYTHONUNBUFFERE` variable